js/src/parse.js

```diff
@@ -88,7 +100,7 @@ function getStringValue(value, token) {
 /**
  * Gets the JavaScript value represented by a JSON token.
  * @param {string} value The text value of the token.
- * @param {Token} token The JSON token to get a value for.
+ * @param {MomoaToken} token The JSON token to get a value for.
  * @returns {*} A number, string, boolean, or `null`. 
```

Comment:
Instead of using `{*}` is better to put the types you mention just next to this: `{number | string | boolean | null}`

Reply:
Ah, nice tip. Thank you.